shell/renderer/renderer_client_base.h

```diff
@@ -86,6 +86,11 @@ class RendererClientBase : public content::ContentRendererClient {
   void RunScriptsAtDocumentStart(content::RenderFrame* render_frame) override;
   void RunScriptsAtDocumentEnd(content::RenderFrame* render_frame) override;
   void RunScriptsAtDocumentIdle(content::RenderFrame* render_frame) override;
+  bool ShouldFork(blink::WebLocalFrame* frame,
```

Comment:
Returning `true` for `ShouldFork` has quite a lot of side effects, and one reason `sandbox` mode was introduced was to avoid that, I don't think this solution can be accepted.

Reply:
so  is there a  good resolution we can  fire 'will-navigation'  in  sandbox mode?